DEPS

```diff
@@ -12,7 +12,7 @@ vars = {
   'chromium_version':
     '84c40395c741fa24ccbd9fc2c5828e2e97472952',
   'node_version':
-    'a86a4a160dc520c61a602c949a32a1bc4c0fc633',
+    'b213b26f0fb642b95561c0f10039886607036d5a',
```

Comment:
link to electron/nodejs pr?

Reply:
Tis just this commit --> https://github.com/electron/node/commit/b213b26f0fb642b95561c0f10039886607036d5a